examples/basics/src/components/Card.astro

```diff
@@ -69,6 +70,7 @@ const { href, title, body } = Astro.props;
 	}
 
 	.link-card:is(:hover, :focus-within) h2 span {
+		will-change: transform;
```

Comment:
This is needed to prevent a weird flash on hover due to rendering shifting to the GPU during the transformation (I assume?)